### PR TITLE
Processing modeler chapter

### PR DESCRIPTION
**Pull request by ghtmtt**
Processing modeler chapter added

### CODE REVIEW

**Comment by ghtmtt** [comment]
Here we are with anchor duplication even if in different files:

* new chapter added with `_loadlayer` anchor: https://github.com/qgis/QGIS-Documentation/pull/2364/files#diff-1992745e3559bc4936105998960d1c69R15
* and existing anchor in `pyqgis` cookbook: https://github.com/qgis/QGIS-Documentation/blob/master/source/docs/pyqgis_developer_cookbook/loadlayer.rst

how should be handle this problem? If this gave error then also other duplicated anchors will stop Travis for compilation.

@rduivenvoorde @DelazJ

**Comment by rduivenvoorde** [comment]
Mmm, as we cannot change the naming machinery of the algorithms, I think the only way to fix/handle this is 'just' renaming the anchor in the cookbook.
But you are right... this is tricky, as: IF we have two algorithms with the same name, we are in trouble...

File: source/docs/user_manual/processing_algs/qgis/modelertools.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+.. _loadlayer:
+
+Load layer into project
+=======================
+Loads a layer to the current project.
+
+Parameters
+..........
+
+``Layer`` [multipleinput]
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/modelertools.rst]: I'm not sure this is a multiple input parameter (same for other alg) or at least I failed to do the selections.

File: source/docs/user_manual/processing_algs/qgis/modelertools.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+.. _renamelayer:
+
+Rename layer
+============
+Renames a layer in the legend.
+
+Parameters
+..........
+
+``Layers`` [multipleinput]
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/modelertools.rst]: `layer` (and not multipleinput, I'm afraid). Don't know which "tag" to use: [any]?

**Pull request merged**